test/Project.toml

```diff
@@ -0,0 +1,24 @@
+[deps]
+BlackBoxOptim = "a134a8b2-14d6-55f6-9291-3336d3ab0209"
```

Comment:
adding this means that the Project.toml should be updated to remove the test deps from there. Also, double check we have a Julia v1.3 or higher lower bound.

Reply:
We got `julia = "1.5"` in the `[compat]` section so it should be fine